bitmap-plasma/app/src/main/jni/plasma.c

```diff
@@ -140,7 +140,7 @@ static uint16_t  palette[PALETTE_SIZE];
 static uint16_t  make565(int red, int green, int blue)
 {
     return (uint16_t)( ((red   << 8) & 0xf800) |
-                       ((green << 2) & 0x03e0) |
+                       ((green << 3) & 0x07e0) |
                        ((blue  >> 3) & 0x001f) );
 }
```

Comment:
nit: Is below more straightforward?
     return static_cast<uint16_t>( (red  & 0xf8) << 8)  |
                       ((green & 0xfc) << 3) |
                         ((blue  & 0xf8) >> 3));